roles/vpn/templates/etc_openvpn_server.conf.j2

```diff
@@ -190,6 +190,7 @@ ifconfig-pool-persist ipp.txt
 ;push "redirect-gateway def1 bypass-dhcp"
 push "redirect-gateway def1"
 push "dhcp-option DNS 10.8.0.1"
+push "dhcp-option DNS 8.8.8.8"
```

Comment:
The 10.8.0.1 is there to allow the openvpn client to use the server's dnsmasq, right? If 8.8.8.8 needs to be added as a default DNS then it should be done in the dnsmasq configuration, I think. I'm not sure it makes sense to add it here.

I did use the VPN at one point quite a bit and I never had dns problems. This makes me think the server dnsmasq was working fine.